DSESwift/DataStorageEngine.swift

```diff
@@ -31,12 +30,12 @@ public class DataStorageEngine{
     // reference to the datasyncer
     var dataSyncer = DataSyncer()
```

Comment:
Do you mean if DataSyncer has whether public, internal or private access level? if so, then yes it has internal access level.